tests/update-ts-references.test.js

```diff
@@ -142,3 +147,52 @@ test('Support lerna', async () => {
     ).toEqual(config);
   });
 });
+
+test('Detect changes with the --check option', async () => {
+  let errorCode = 0;
+  try {
+    await execSh('npx update-ts-references --check', {
+      stdio: null,
+      cwd: rootFolderYarnCheck,
+    });
+  } catch (e) {
+    errorCode = e.code;
+  }
+
+  expect(errorCode).toBe(6);
```

Comment:
What is the meaning of the `6`? Is it a count of 6 errors and each error stands for one change found while checking? Or is it a code for one error case?

Reply:
It would require 6 tsconfig.json to update for that scenario.

Reply:
Thanks for explaining. I got it. I found the code where this information is generated:  `process.exit(changesCount);` inside of the `src/index.js`. 👍